.github/ISSUE_TEMPLATE/keycloak_realm_request.md

```diff
@@ -11,33 +11,47 @@ assignees: caggles, ShellyXueHan
 **Are you the product owner or project admin/team lead?**
 If not, you will need to ask one of those two people to make this request for you. We can't process realm requests except from individuals in these roles.
 
-**Do you have an IDIR account?**
-We use IDIR authentication for realm admin users, so you *must* have one in order to become a realm admin.  
+**Does the Admin User have an IDIR account?**
+We use IDIR authentication for Realm Admin Users, so make sure there's a *valid IDIR* for the assigned realm admin.  
 
 **Are you requesting for Identity Provider Update?**
-If so, you need to be the realm admin. Then proceed to Step 2.
+If so, you need to be the realm admin. Then proceed to Step 3.
 
 
 ## Step 1
-Ask the Realm Admin User to login to each of the following links using the **IDIR credentials**. There will show a "_Forbidden_" message, but this is expected behaviour and will actually create the initial account on KeyCloak.
+Have you had an onboarding meeting yet? If not, make sure you are available on [RocketChat](https://chat.pathfinder.gov.bc.ca/). We will contact you soon to book a meeting.
```

Comment:
This feels ambiguous. We tell them to goto rocketchat but is there a specifc channel they need to goto?

Reply:
no specific channel, but more like to make sure the requester is reachable on RC so that me/Cailey/Clecio will be able to contact them to initiate the meeting. Originally I was thinking of having the conversation in the ticket, but looks like what Olena has tried out is working all good.

Reply:
So i'm a bit confused how do they know to speak to you,cailey or clecio in the first place? How do you know to speak to them?

Reply:
kkk got what you meant. I'll update it